config/routes.rb

```diff
@@ -0,0 +1,61 @@
+Rails.application.routes.draw do
+  # get 'home/index'
+  # get "home" => "home#index"
+  # get "/" => "home#index"
+  root  'home#index'
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.